README.md

```diff
@@ -4,16 +4,38 @@ Node.js NAN C++ NuoDB Driver
 
 The node-nuodb add-on for Node.js powers high performance NuoDB Database applications.
 
-Use node-nuodb to connect Node.js 8 and 10 to NuoDB Database.
+Use node-nuodb to connect Node.js to a NuoDB Database.
 
 node-nuodb supports basic features of the NuoDB Database and NuoDB C++ client.
 
+## Dependencies
+
+Node 16.16.0+
```

Comment:
As far as I know this works with 16.13.2, and as far as I know that is the only version we support.  So we should change this number accordingly.

Reply:
Happy to make that change.